examples/tutorials/Part 10 - Secure Deep Learning Classification.ipynb

```diff
@@ -49,7 +49,7 @@
   },
```

Comment:
<p>We don't call the conv ever so is it needed?</p>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2129/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Sorry, debugging code for some other stuff I was trying.</p>